### ISSUE TITLE
Angular output contains superflous global.d.ts

### ISSUE THREAD

**Issue by kylecordes** [BuilderIO/mitosis#501]
### Currently

The Angular output includes a file `global.d.ts`, containing JavaScript output from a bundling process, starting with:

```
var __create = Object.create;
var __defProp = Object.defineProperty;
var __getOwnPropDesc = Object.getOwnPropertyDescriptor;
...
```

When this file is seen by Angular tooling (for example, Angular CLI), building the Angular app fails.

### Expected

Angular output should contain only the exact files/contents that are accepted by commonly used Angular tools - no superflous files.

**Comment by PatrickJS** [comment]
This is with the CLI right?

**Comment by kylecordes** [comment]
Yes, with `mitosis build`, which I assume will be the main way "the masses" use Mitosis when it takes over the industry.

**Comment by kylecordes** [comment]
(I'm working on E2E tests, which use Mitosis completely via the CLI - that's the most end-to-end I think. If the CLI produces correct results, all the code throughout works.)
